src/main/java/edu/oakland/production/middleware01/MiddlewareGisInterfaceImplementation.java

```diff
@@ -0,0 +1,45 @@
+package edu.oakland.production.middleware01;
+
+import edu.oakland.helper.admin.Satellite;
+import edu.oakland.production.middleware01.MiddlewareGisInterface;
```

Comment:
While it's building without it being required, I'd feel better if we imported the `MiddlewareGisProcessor` here as well.
```suggestion
import edu.oakland.production.middleware01.MiddlewareGisInterface;
import edu.oakland.production.middleware01.MiddlewareGisProcessor;
```